lib/console_web/controllers/router/device_controller.ex

```diff
@@ -425,4 +425,24 @@ defmodule ConsoleWeb.Router.DeviceController do
         end
     end
   end
+
+  def update_devices_in_xor_filter(conn, %{"added" => added_device_ids, "removed" => removed_device_ids}) do
+    result = Ecto.Multi.new()
+      |> Ecto.Multi.run(:added_devices, fn _repo, _ ->
+        with {:ok, devices} <- Devices.update_in_xor_filter(added_device_ids, true) do
+          {:ok, devices}
+        end
+      end)
+      |> Ecto.Multi.run(:removed_devices, fn _repo, _ ->
```

Comment:
I think we dont need to remove any devices. Michael is just sending them to cover all bases

Reply:
oh gotcha. will remove